### ISSUE TITLE
Error in MenuBar when Accessibility has not been turned on for CP

### ISSUE THREAD

**Issue by latenitefilms** [fcpxhacks/fcpxhacks#407]
The below error comes up when going through the Welcome Screen.

```
CommandPost v0.79

macOS Version:                  10.12.3
CommandPost Locale:             en
Current Keyboard Layout:        U.S.
Final Cut Pro Path:             /Applications/Final Cut Pro.app
Final Cut Pro Version:          10.3.2
Final Cut Pro Language:         en
Loaded from Bundle:             true
Developer Mode:                 true

2017-03-26 20:11:16: 2017-03-26 20:11:15: -- Loading /Users/latenitechris/Documents/Github/CommandPost/src/extensions/cp/init.lua
2017-03-26 20:11:15: -- Loading extension: uielement
2017-03-26 20:11:15: -- Loading extension: window
2017-03-26 20:11:15: 20:11:15    menubar:     Loaded menu map from '/Users/latenitechris/Documents/Github/CommandPost/src/extensions/cp/finalcutpro/menumap.json'
2017-03-26 20:11:15:          finalcutpr:     Reloading Final Cut Pro Preferences.
2017-03-26 20:11:16: 20:11:16         cp:     Loading Plugins:
2017-03-26 20:11:16:             plugins:     Loaded plugin: core.action.manager
2017-03-26 20:11:16:                          Loaded plugin: core.commands.commandaction
2017-03-26 20:11:16:                          Loaded plugin: core.commands.global
2017-03-26 20:11:16:                          Loaded plugin: core.helpandsupport.credits
2017-03-26 20:11:16:                          Loaded plugin: core.helpandsupport.feedback
2017-03-26 20:11:16:                          Loaded plugin: core.helpandsupport.userguide
2017-03-26 20:11:16:                          Loaded plugin: core.language
2017-03-26 20:11:16:                          Loaded plugin: core.menu.bottom
2017-03-26 20:11:16:                          Loaded plugin: core.menu.helpandsupport
2017-03-26 20:11:16:                          Loaded plugin: core.menu.manager
2017-03-26 20:11:16:                          Loaded plugin: core.menu.top
2017-03-26 20:11:16:                          Loaded plugin: core.preferences.general
2017-03-26 20:11:16:                          Loaded plugin: core.preferences.manager
2017-03-26 20:11:16:                          Loaded plugin: core.preferences.panels.general
2017-03-26 20:11:16:                          Loaded plugin: core.preferences.panels.menubar
2017-03-26 20:11:16:                          Loaded plugin: core.preferences.panels.plugins
2017-03-26 20:11:16:                          Loaded plugin: core.preferences.panels.shortcuts
2017-03-26 20:11:16:                          Loaded plugin: core.preferences.updates
2017-03-26 20:11:16:                          Loaded plugin: core.quit
2017-03-26 20:11:16:                          Loaded plugin: core.welcome.manager
2017-03-26 20:11:16:                          Loaded plugin: core.welcome.panels.accessibility
2017-03-26 20:11:16:                          Loaded plugin: core.welcome.panels.complete
2017-03-26 20:11:16:                          Loaded plugin: core.welcome.panels.intro
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.browser.addnote
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.browser.keywords
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.browser.playhead
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.clipboard.history
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.clipboard.manager
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.clipboard.shared
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.commands
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.console
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.export.batch
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.fullscreen.shortcuts
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.hacks.backupinterval
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.hacks.movingmarkers
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.hacks.playbackrendering
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.hacks.shortcuts
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.hacks.smartcollectionslabel
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.hacks.timecodeoverlay
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.hud
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.import.ignorecard
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.import.preferences
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.language
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.menu.administrator.advancedfeatures
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.menu.administrator
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.menu.clipboard
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.menu.mediaimport
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.menu.menuaction
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.menu.proxyicon
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.menu.timeline.assignshortcuts
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.menu.timeline.highlightplayhead
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.menu.timeline
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.menu.tools.notifications
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.menu.tools
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.notifications.imessage
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.notifications.manager
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.notifications.prowl
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.open
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.os.touchbar
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.os.voice
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.preferences.panels.finalcutpro
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.preferences.scanfinalcutpro
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.sharing.xml
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.timeline.colorboard
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.timeline.effects
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.timeline.generators
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.timeline.height
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.timeline.lanes
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.timeline.matchframe
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.timeline.movetoplayhead
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.timeline.multicam
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.timeline.playback
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.timeline.playhead
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.timeline.preferences
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.timeline.selectalltimelineclips
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.timeline.titles
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.timeline.transitions
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.watchers.preferences
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.watchers.version
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.welcome.panels.commandset
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.welcome.panels.finalcutpromissing
2017-03-26 20:11:16:                          Loaded plugin: finalcutpro.welcome.panels.scanfinalcutpro
2017-03-26 20:11:16:                          Initialised plugin: core.action.manager
2017-03-26 20:11:16:                          Initialised plugin: core.commands.commandaction
2017-03-26 20:11:16:                          Initialised plugin: core.commands.global
2017-03-26 20:11:16:                          Initialised plugin: core.menu.manager
2017-03-26 20:11:16:                          Initialised plugin: core.menu.bottom
2017-03-26 20:11:16:                          Initialised plugin: core.menu.helpandsupport
2017-03-26 20:11:16:                          Initialised plugin: core.helpandsupport.credits
2017-03-26 20:11:16:                          Initialised plugin: core.helpandsupport.feedback
2017-03-26 20:11:16:                          Initialised plugin: core.helpandsupport.userguide
2017-03-26 20:11:16:                          Initialised plugin: core.preferences.manager
2017-03-26 20:11:16:                          Initialised plugin: core.preferences.panels.general
2017-03-26 20:11:16:                          Initialised plugin: core.language
2017-03-26 20:11:16:                          Initialised plugin: core.menu.top
2017-03-26 20:11:16:                          Initialised plugin: core.preferences.panels.menubar
2017-03-26 20:11:16:                          Initialised plugin: core.preferences.general
2017-03-26 20:11:16:                          Initialised plugin: core.preferences.panels.plugins
2017-03-26 20:11:16:                          Initialised plugin: finalcutpro.commands
2017-03-26 20:11:16:                          Initialised plugin: core.preferences.panels.shortcuts
2017-03-26 20:11:16:                          Initialised plugin: core.preferences.updates
2017-03-26 20:11:16:                          Initialised plugin: core.quit
2017-03-26 20:11:16:                          Initialised plugin: finalcutpro.preferences.panels.finalcutpro
2017-03-26 20:11:16:                          Initialised plugin: finalcutpro.hacks.shortcuts
2017-03-26 20:11:16:                          Initialised plugin: core.welcome.manager
2017-03-26 20:11:16:                          Initialised plugin: core.welcome.panels.accessibility
2017-03-26 20:11:16:                          Initialised plugin: core.welcome.panels.complete
2017-03-26 20:11:16:                          Initialised plugin: core.welcome.panels.intro
2017-03-26 20:11:16:                          Initialised plugin: finalcutpro.browser.addnote
2017-03-26 20:11:16:                          Initialised plugin: finalcutpro.browser.keywords
2017-03-26 20:11:16:                          Initialised plugin: finalcutpro.menu.timeline
2017-03-26 20:11:16:                          Initialised plugin: finalcutpro.menu.timeline.highlightplayhead
2017-03-26 20:11:16:                          Initialised plugin: finalcutpro.browser.playhead
2017-03-26 20:11:16:                          Initialised plugin: finalcutpro.clipboard.manager
2017-03-26 20:11:16:                          Initialised plugin: finalcutpro.menu.clipboard
2017-03-26 20:11:16:                          Initialised plugin: finalcutpro.clipboard.history
2017-03-26 20:11:17: 20:11:17                 Initialised plugin: finalcutpro.clipboard.shared
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.menu.tools
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.console
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.export.batch
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.fullscreen.shortcuts
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.menu.administrator
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.menu.administrator.advancedfeatures
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.hacks.backupinterval
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.hacks.movingmarkers
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.hacks.playbackrendering
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.hacks.smartcollectionslabel
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.hacks.timecodeoverlay
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.sharing.xml
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.hud
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.menu.mediaimport
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.import.ignorecard
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.import.preferences
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.language
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.menu.menuaction
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.menu.proxyicon
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.menu.timeline.assignshortcuts
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.menu.tools.notifications
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.notifications.manager
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.notifications.imessage
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.notifications.prowl
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.open
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.os.touchbar
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.os.voice
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.timeline.transitions
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.timeline.titles
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.timeline.effects
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.timeline.generators
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.preferences.scanfinalcutpro
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.timeline.colorboard
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.timeline.height
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.timeline.lanes
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.timeline.matchframe
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.timeline.movetoplayhead
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.timeline.multicam
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.timeline.playback
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.timeline.playhead
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.timeline.preferences
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.timeline.selectalltimelineclips
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.watchers.preferences
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.watchers.version
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.welcome.panels.commandset
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.welcome.panels.finalcutpromissing
2017-03-26 20:11:17:                          Initialised plugin: finalcutpro.welcome.panels.scanfinalcutpro
2017-03-26 20:11:17: -- Done.
2017-03-26 20:11:20: ********
2017-03-26 20:11:20: 20:11:20 ERROR:   LuaSkin: hs.chooser:choices error - ...ub/CommandPost/src/extensions/cp/finalcutpro/MenuBar.lua:211: attempt to index a nil value (local 'menu')
stack traceback:
	...ub/CommandPost/src/extensions/cp/finalcutpro/MenuBar.lua:211: in function 'cp.finalcutpro.MenuBar._visitMenuItems'
	(...tail calls...)
	.../CommandPost/src/plugins/finalcutpro/menu/menuaction.lua:121: in field 'choices'
	...s/Github/CommandPost/src/plugins/core/action/manager.lua:343: in field '_findChoices'
	...s/Github/CommandPost/src/plugins/core/action/manager.lua:334: in function <...s/Github/CommandPost/src/plugins/core/action/manager.lua:332>
	(...tail calls...)
	[C]: in method 'refreshChoicesCallback'
	.../CommandPost/src/plugins/finalcutpro/console/console.lua:142: in function <.../CommandPost/src/plugins/finalcutpro/console/console.lua:141>
2017-03-26 20:11:20: ********
2017-03-26 20:11:20: ********
2017-03-26 20:11:20: 20:11:20 ERROR:   LuaSkin: hs.chooser:choices error - ...ub/CommandPost/src/extensions/cp/finalcutpro/MenuBar.lua:211: attempt to index a nil value (local 'menu')
stack traceback:
	...ub/CommandPost/src/extensions/cp/finalcutpro/MenuBar.lua:211: in function 'cp.finalcutpro.MenuBar._visitMenuItems'
	(...tail calls...)
	.../CommandPost/src/plugins/finalcutpro/menu/menuaction.lua:121: in field 'choices'
	...s/Github/CommandPost/src/plugins/core/action/manager.lua:343: in field '_findChoices'
	...s/Github/CommandPost/src/plugins/core/action/manager.lua:327: in function <...s/Github/CommandPost/src/plugins/core/action/manager.lua:325>
	[C]: in method 'refreshChoicesCallback'
	.../CommandPost/src/plugins/finalcutpro/console/console.lua:143: in function <.../CommandPost/src/plugins/finalcutpro/console/console.lua:141>
2017-03-26 20:11:20: ********
2017-03-26 20:11:26: ********
2017-03-26 20:11:26: 20:11:26 ERROR:   wfilter: app universalAccessAuthWarn is not registered!
2017-03-26 20:11:26: ********
2017-03-26 20:11:28: ********
2017-03-26 20:11:28: 20:11:28 ERROR:   wfilter: app System Preferences is not registered!
2017-03-26 20:11:28: ********
2017-03-26 20:12:12: 20:12:12 ** Warning:   LuaSkin: hs.menubar:_insertStatusItem:withPriority: unavailable in this version of OS X
2017-03-26 20:12:12:           shortcuts:     Initialising shortcuts...

```

**Comment by randomeizer** [comment]
This happens when CP has not been given Accessibility permissions yet. It should handle the case that the menu comes back as 'nil'.

**Comment by randomeizer** [comment]
Fixed in #306
